script/testing/jdbc/src/PelotonTest.java

```diff
@@ -14,7 +14,7 @@
 import org.postgresql.util.*;
 
 public class PelotonTest {
-  private final String url = "jdbc:postgresql://localhost:54321/";
+  private final String url = "jdbc:postgresql://localhost:5432/";
```

Comment:
We should have a port number consistent with the shell script
https://github.com/cmu-db/peloton/blob/master/script/validators/jdbc_validator.py#L34

(Usually psql already occupies port 5432 on my local machine which causes the validator to fail so I changed it to 54321, but that's a minor point).